Sources/ManageMaintenance.php

```diff
@@ -1248,9 +1248,12 @@ function AdminBoardRecount()
 
 	// Update the latest message of each board.
 	$request = $smcFunc['db_query']('', '
-		SELECT m.id_board, MAX(m.id_msg) AS local_last_msg
+		SELECT m.id_board,
+			(SELECT MAX(id_msg)
```

Comment:
Why use of subquery which can be not rewrittrm by old mysql Version?

Reply:
Subqueries are supported by all of SMF's supported versions of mysql.